Firmware/MVP/BATTSCI.ino

```diff
@@ -79,8 +79,37 @@ void BATTSCI_sendFrames(struct packetTypes METSCI_Packets, uint8_t stackVoltage,
       frameSum_87 += BATTSCI_writeByte( 0x87 );                                            //Never changes
       frameSum_87 += BATTSCI_writeByte( 0x40 );                                            //Never changes
       frameSum_87 += BATTSCI_writeByte( (stackVoltage >> 1) );                             //Half battery voltage (e.g. 0x40 = d64 = 128 V
-      frameSum_87 += BATTSCI_writeByte( 0x15 );                                            //Battery SoC (upper byte)
-      frameSum_87 += BATTSCI_writeByte( 0x6F );                                            //Battery SoC (lower byte)
+//      frameSum_87 += BATTSCI_writeByte( 0x16 );                                            //Battery SoC (upper byte)
+//      frameSum_87 += BATTSCI_writeByte( 0x20 );                                            //Battery SoC (lower byte)
+      switch (desiredBehaviourFlag) {
```

Comment:
Change to switch(stackVoltage){...
//add your conditional logic inside BATTSCI_sendFrames (e.g. "if (stackVoltage > 180)...")